### ISSUE TITLE
Album not included in collection

### ISSUE THREAD

**Issue by exptom** [strawberrymusicplayer/strawberry#843]
Strawberry is set to automatically scan an NFS share that I have mounted on my computer that is exported from my NAS.
This generally works fine but there is one particular album that I cannot get strawberry to include in the collection.

**Describe the bug**
I have run with --verbose and captured the logs when scanning this album. 
Unlike my other albums I don't see any log entries for "Reading tags from" for the tracks of this album - but I also don't see any error messages relating to invalid tags.

```
16:05:18.818 DEBUG _MessageReplyBase:31             Waiting on ID 6100
16:05:18.818 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/04 The Butcher.flac"
16:05:18.819 DEBUG MessageReply<MessageType>:90     Releasing ID 6100 (finished)
16:05:18.820 DEBUG _MessageReplyBase:33             Acquired ID 6100
16:05:18.820 DEBUG _MessageReplyBase:31             Waiting on ID 6101
16:05:18.820 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/01 Sincerely Ichabod.flac"
16:05:18.821 DEBUG MessageReply<MessageType>:90     Releasing ID 6101 (finished)
16:05:18.821 DEBUG _MessageReplyBase:33             Acquired ID 6101
16:05:18.822 DEBUG _MessageReplyBase:31             Waiting on ID 6102
16:05:18.822 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/12 The Spy Hunter.flac"
16:05:18.823 DEBUG MessageReply<MessageType>:90     Releasing ID 6102 (finished)
16:05:18.823 DEBUG _MessageReplyBase:33             Acquired ID 6102
16:05:18.823 DEBUG _MessageReplyBase:31             Waiting on ID 6103
16:05:18.823 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/02 Safe Haven.flac"
16:05:18.825 DEBUG MessageReply<MessageType>:90     Releasing ID 6103 (finished)
16:05:18.825 DEBUG _MessageReplyBase:33             Acquired ID 6103
16:05:18.825 DEBUG _MessageReplyBase:31             Waiting on ID 6104
16:05:18.825 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/05 Last Meal.flac"
16:05:18.828 DEBUG MessageReply<MessageType>:90     Releasing ID 6104 (finished)
16:05:18.828 DEBUG _MessageReplyBase:33             Acquired ID 6104
16:05:18.829 DEBUG _MessageReplyBase:31             Waiting on ID 6105
16:05:18.829 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/10 Destroyer.flac"
16:05:18.831 DEBUG MessageReply<MessageType>:90     Releasing ID 6105 (finished)
16:05:18.831 DEBUG _MessageReplyBase:33             Acquired ID 6105
16:05:18.831 DEBUG _MessageReplyBase:31             Waiting on ID 6106
16:05:18.831 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/03 Oblivion.flac"
16:05:18.833 DEBUG MessageReply<MessageType>:90     Releasing ID 6106 (finished)
16:05:18.833 DEBUG _MessageReplyBase:33             Acquired ID 6106
16:05:18.834 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/08 Illuminate.flac"
16:05:18.834 DEBUG _MessageReplyBase:31             Waiting on ID 6107
16:05:18.837 DEBUG MessageReply<MessageType>:90     Releasing ID 6107 (finished)
16:05:18.837 DEBUG _MessageReplyBase:33             Acquired ID 6107
16:05:18.837 DEBUG _MessageReplyBase:31             Waiting on ID 6108
16:05:18.837 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/07 SMC.flac"
16:05:18.838 DEBUG MessageReply<MessageType>:90     Releasing ID 6108 (finished)
16:05:18.838 DEBUG _MessageReplyBase:33             Acquired ID 6108
16:05:18.838 DEBUG _MessageReplyBase:31             Waiting on ID 6109
16:05:18.838 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/09 Evil.flac"
16:05:18.839 DEBUG MessageReply<MessageType>:90     Releasing ID 6109 (finished)
16:05:18.839 DEBUG _MessageReplyBase:33             Acquired ID 6109
16:05:18.840 DEBUG _MessageReplyBase:31             Waiting on ID 6110
16:05:18.840 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/06 Me vs. Me.flac"
16:05:18.841 DEBUG MessageReply<MessageType>:90     Releasing ID 6110 (finished)
16:05:18.841 DEBUG _MessageReplyBase:33             Acquired ID 6110
16:05:18.841 DEBUG _MessageReplyBase:31             Waiting on ID 6111
16:05:18.841 DEBUG TagReaderTagLib:147              Checking for valid file "/home/tom/Mounts/music/Project 86/15. Live_/11 Stein's Theme.flac"
16:05:18.843 DEBUG MessageReply<MessageType>:90     Releasing ID 6111 (finished)
16:05:18.843 DEBUG _MessageReplyBase:33             Acquired ID 6111
```

I have also tested all the flac files `flac -t` and they report as `ok`:
```
flac 1.3.3
Copyright (C) 2000-2009  Josh Coalson, 2011-2016  Xiph.Org Foundation
flac comes with ABSOLUTELY NO WARRANTY.  This is free software, and you are
welcome to redistribute it under certain conditions.  Type `flac' for details.

01 Sincerely Ichabod.flac: ok                    
02 Safe Haven.flac: ok                    
03 Oblivion.flac: ok                    
04 The Butcher.flac: ok                    
05 Last Meal.flac: ok                    
06 Me vs. Me.flac: ok                    
07 SMC.flac: ok                    
08 Illuminate.flac: ok                    
09 Evil.flac: ok                    
10 Destroyer.flac: ok                    
11 Stein's Theme.flac: ok                    
12 The Spy Hunter.flac: ok  
```

**Expected behavior**
Album included in my music collection

**System Information:**
 - Operating system: Ubuntu 21.04
 - Strawberry Version: 1.0

**Comment by jonaski** [comment]
Does it work if you add the files by dragging and dropping from the file manager, or with the playlist / add file menu?

**Comment by exptom** [comment]
If I do "Playlist -> Add Folder" it does play them but it shows the files as "unknown" file type until I actually start playing a track. When the track finishes it goes back to being "unknown" file type.
![image](https://user-images.githubusercontent.com/4302032/144311106-03f5bccb-dea7-487f-88b5-bc99d084e3dc.png)

**Comment by jonaski** [comment]
Strange, can you send me one of the files on e-mail?
My e-mail is listed on my GH profile page.

**Comment by jonaski** [comment]
The file you sent works fine here.

![image](https://user-images.githubusercontent.com/10343810/144464177-861c02bc-3a6a-4f83-b5d5-37992daddceb.png)

This is probably a bug in taglib that has already been fixed in the latest release 1.12
According to https://packages.ubuntu.com/hirsute/libtag1-dev Ubuntu (all version) still uses TagLib 1.11.1, that's a 5 year old release.
So you should report this bug to Ubuntu.

**Comment by exptom** [comment]
Thank you so much @jonaski for investigating! I've managed to find a workaround for my issue after investigating TagLib 1.12.

For anyone else who stumbles on this later:
* TagLib 1.12 fixes a bug "Fixed reading FLAC files with zero-sized seektables."
* As 1.12 is not yet in ubuntu I have worked around the issue by removing the zero-sized seektable from my problematic tracks using the `metaflac` command.
`metaflac --remove --block-type=SEEKTABLE <path/to/file.flac>`
